internal/job/checkout.go

```diff
@@ -217,6 +217,11 @@ func (e *Executor) CheckoutPhase(ctx context.Context) error {
 		}
 	}
 
+	err = e.sendCommitToBuildkite(ctx)
+	if err != nil {
+		return err
```

Comment:
Are you satisfied that this won't be a breaking change for anyone? I wonder if there are customers out there who have modified their checkout phase to such an extent that this won't resolve – in which case logging this as a warning rather than erroring out might be a good idea? I'm not sure - just want to make sure we cover the bases